jupyter_nbconvert/_version.py

```diff
@@ -1,2 +1,41 @@
-version_info = (4, 0, 0, 'dev')
-__version__ = '.'.join(map(str, version_info))
```

Comment:
I think I'd rather keep this file small and simple, and just add `_version_extra` to check for the 'dev' tag.